src/EnergyPlus/SystemReports.hh

```diff
@@ -331,8 +331,6 @@ struct SystemReportsData : BaseGlobalStruct
     Array1D<Real64> SysHCCompSteam;
     Array1D<Real64> SysDomesticH2O;
 
-    Array1D<Real64> ZoneOAMassFlow;               // zone mech vent mass flow rate {kg/s}
-    Array1D<Real64> ZoneOAMass;                   // zone mech vent total mass for time {kg}
     Array1D<Real64> ZoneOAVolFlowStdRho;          // zone mech vent volume flow rate at standard density {m3/s}
     Array1D<Real64> ZoneOAVolStdRho;              // zone mech vent total volume OA at standard density {m3}
     Array1D<Real64> ZoneOAVolFlowCrntRho;         // zone mech vent volume flow rate at current density {m3/s}
```

Comment:
SystemReports has many `Array1D` that are used primarily in one function, so I was hoping to make them local to `ReportMaxVentilationLoads` but many of them are also referenced as output variables, so they need to remain in `state`.

Reply:
📌 These should reorganized into two, three, or four arrays of `struct`s rather than 67 (not a made up number) of arrays of scalars.  

📌 This module could also use a `struct AirLoopLocation` that is analogous to the `struct PlantLocation` we are using now.

Reply:
Is there a limit to how large a single struct should be? I think everything here could be in either one struct by zone and another by airloop.